wicket-jquery-ui-samples/src/main/java/com/googlecode/wicket/jquery/ui/samples/pages/kendo/menu/KendoContextMenuPage.html

```diff
@@ -13,6 +13,12 @@
 	<div wicket:id="menu"></div>
 	<br/>
 	<div wicket:id="feedback"></div>
+
+	<wicket:fragment wicket:id="moveToPosition">
+		<form wicket:id="form" onclick="var $evt=jQuery.event.fix(event);if($evt.target.nodeName !== 'BUTTON'){$evt.stopPropagation()}">
```

Comment:
- I understand the purpose of this, but I think it is not straightforward for the user willing to achieve the same usecase... This is just a thought but, what about a special Form which can handle this (ie: Menu.InnerForm, MenuForm, etc)
- `jQuery.event.fix`. In the sample you can use `$` (just fyi)